terraform_files/vsphere/variables-vsphere.tf

```diff
@@ -0,0 +1,91 @@
+//////
+// vSphere variables
+//////
+
+variable "vsphere_vcenter" {
+  type        = string
+  description = "vSphere vcenter server ip address or fqdn (vCenter server name for vSphere API operations)"
+}
+
+variable "vsphere_username" {
+  type        = string
+  description = "vSphere vcenter server username"
+}
+
+variable "vsphere_password" {
+  type        = string
+  description = "vSphere vcenter server username"
+}
```

Comment:
Rearrange these variables into groups. It's not clear what's mandatory for vSphere login and what's used for other APIs

Reply:
It's still unclear to me what's required for API operations and what's only for login.
In addition, `iso_download_path` shouldn't be under `vSphere variables` category.